docs/UserGuide.md

```diff
@@ -316,6 +316,20 @@ you fully understand each command and their usage.
 --------------------------------------------------------------------------------------------------------------------
 ## Commands
 
+### Command Summary
+
+| Action                                        | Description                              | Format |
+|-----------------------------------------------|------------------------------------------|------------------------------------------|
+| [add](#adding-an-entry-add)                   | Adds an Internship.                      | `add /com COMPANY_NAME /desc DESCRIPTION /status STATUS /poc CONTACT_NAME /email CONTACT_EMAIL /phone CONTACT_NUMBER /loc LOCATION_ENUM /role ROLE` |
+| [delete](#deleting-an-internship-delete)      | Removes a Internship.                    | `delete INDEX` |
+| [list](#listing-all-internships-list)         | Removes a Internship.                    | `list` |
+| [edit](#editing-an-internship-edit)           | Modifies an existing Internship.         | `edit INDEX [/com COMPANY_NAME] [/poc CONTACT_NAME] [/email CONTACT_EMAIL] [/phone CONTACT_NUMBER] [/loc LOCATION_ENUM] [/status STATUS] [/desc DESCRIPTION] [/role ROLE` |
+| [addremark](#adding-a-remark-addremark)       | Adds a remark to an existing Internship. | `addremark INDEX [/remark REMARK]` |
+| [clear](#clearing-all-internships-clear)      | Removes all Internships from the deck.   | `clear` |
+| [find](#finding-internships-by-keywords-find) | Finds an Internship by keywords.         |`find MODE [/com COMPANY_NAME_KEYWORDS] [/poc CONTACT_NAME_KEYWORDS] [/loc LOCATION_KEYWORDS] [/status STATUS_KEYWORDS] [/desc DESCRIPTION_KEYWORDS] [/role ROLE_KEYWORDS]`|
+| [exit](#exiting-the-program-exit)             | Exits and closes the application.        | `exit` |
+
+
```

Comment:
Shifted this above to right after Quick Start (as discussed in the call).
Should we shift Field Summary up here as well?